cpu/atxmega/ldscripts_compat/README.md

```diff
@@ -0,0 +1,8 @@
+avr linkerscript
+================
+
+Starting from avr-binutils 2.26, the linker defines symbol that can be
+overridden to configure ROM and RAM length.
+
+This ldscript is used for supporting older versions on avr-binutils like on
+ubuntu xenial.
```

Comment:
IMO we should drop these compatibility linker scripts. I don't know anyone still using `xenial`, so those scripts will go largely untested. IMO it is enough to just document that at least binutils in version 2.26 is required.

Reply:
Agreed!